src/index.html

```diff
@@ -11,7 +11,165 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form class="personal_info">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label for="surname">Surname:
+          <input type="text"
+              autocomplete="false"
+              id="surname"
+              name="surname"
+            >
+        </label>
+
+        <label for="name" class="field">Name:
```

Comment:
Перенось текст на новий рядок для кращої читабельності
```suggestion
        <label for="name" class="field">
          Name:
```